From 06e07df001e196a47041bcea65b23f3039e005fa Mon Sep 17 00:00:00 2001
From: Andrey Borzenkov <arvidjaar@gmail.com>
Date: Sat, 19 Feb 2011 09:57:55 +0300
Subject: [PATCH] fix c0a82e271e2730159f042ee7d7fc4aca2e08d28a

Let inst_key_val usage agree with above patch :)

Also UNICODE is rather global console property, not font specific
(and if anything, is rather keyboard specific). Let it be just
vconsole.unicode

Signed-off-by: Andrey Borzenkov <arvidjaar@gmail.com>
Cc: initramfs@vger.kernel.org
---
 modules.d/10i18n/parse-i18n.sh |   16 ++++++++--------
 1 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/modules.d/10i18n/parse-i18n.sh b/modules.d/10i18n/parse-i18n.sh
index 61280c0..73dff6e 100755
--- a/modules.d/10i18n/parse-i18n.sh
+++ b/modules.d/10i18n/parse-i18n.sh
@@ -19,15 +19,15 @@ inst_key_val() {
     unset _value
 }
 
-inst_key_val /etc/vconsole.conf '' KEYMAP      vconsole.keymap      KEYTABLE
-inst_key_val /etc/vconsole.conf '' FONT        vconsole.font        SYSFONT 
-inst_key_val /etc/vconsole.conf '' FONT_MAP    vconsole.font.map    CONTRANS 
-inst_key_val /etc/vconsole.conf '' FONT_UNIMAP vconsole.font.unimap UNIMAP 
-inst_key_val /etc/vconsole.conf 1  UNICODE     vconsole.font.unicode
-inst_key_val /etc/vconsole.conf '' EXT_KEYMAP  vconsole.keymap.ext
+inst_key_val '' /etc/vconsole.conf KEYMAP      vconsole.keymap      KEYTABLE
+inst_key_val '' /etc/vconsole.conf FONT        vconsole.font        SYSFONT 
+inst_key_val '' /etc/vconsole.conf FONT_MAP    vconsole.font.map    CONTRANS 
+inst_key_val '' /etc/vconsole.conf FONT_UNIMAP vconsole.font.unimap UNIMAP 
+inst_key_val 1  /etc/vconsole.conf UNICODE     vconsole.unicode
+inst_key_val '' /etc/vconsole.conf EXT_KEYMAP  vconsole.keymap.ext
 
-inst_key_val /etc/locale.conf   LANG   locale.LANG
-inst_key_val /etc/locale.conf   LC_ALL locale.LC_ALL
+inst_key_val '' /etc/locale.conf   LANG        locale.LANG
+inst_key_val '' /etc/locale.conf   LC_ALL      locale.LC_ALL
 
 if [ -f /etc/locale.conf ]; then
     . /etc/locale.conf
